realm/realm-library/src/main/cpp/io_realm_RealmFileUserStore.cpp

```diff
@@ -23,20 +23,37 @@
 
 using namespace realm;
 
-static const char* ERR_NO_LOGGED_IN_USER = "No user logged in yet.";
 static const char* ERR_COULD_NOT_ALLOCATE_MEMORY = "Could not allocate memory to return all users.";
 
+static jstring
+to_user_string_or_null (JNIEnv *env, const std::shared_ptr<SyncUser>& user)
```

Comment:
I guess CI will fail :P you might need a function declaration before the definition.